charts/rancher-monitoring/v0.0.1/charts/grafana/templates/dashboards-configmap.yaml

```diff
@@ -17,7 +17,7 @@ data:
       "access": "proxy",
       "basicAuth": false,
       "editable": false,
-      "isDefault": true,
+      "isDefault:": true,
```

Comment:
this looks like youre putting the `:` back inside the quotes. do you want to do that?